### PR TITLE
Update closure-util and openlayers to the latest version

### PR DESCRIPTION
**Pull request by fredj**
work in progress

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -51,7 +51,7 @@
     "less": "2.6.0",
     "less-plugin-autoprefix": "1.5.1",
     "nomnom": "1.8.1",
-    "openlayers": "openlayers/ol3#82f6fb9",
+    "openlayers": "openlayers/ol3#d5fb6dd",
```
Comment (sbrunner) [inline package.json]: Not the real version number?

**Comment by fredj** [comment]
Compiler error:
```
./node_modules/.bin/closure-util build .build/app-mobile.json contribs/gmf/build/mobile.closure.js
info closure-util Reading build config
info closure-util Getting Closure dependencies
info closure-util Compiling 417 sources
ERR! compile contribs/gmf/apps/mobile/js/controller.js:10: ERROR - namespace "app.MobileController" cannot be provided twice
ERR! compile goog.provide('app.MobileController');
ERR! compile ^
ERR! compile 
ERR! compile 
ERR! compile contribs/gmf/apps/mobile/js/controller.js:11: ERROR - namespace "app_mobile" cannot be provided twice
ERR! compile goog.provide('app_mobile');
ERR! compile ^
ERR! compile 
ERR! compile 
ERR! compile 2 error(s), 
ERR! compile 0 warning(s)
ERR! compile 
ERR! closure-util Process exited with non-zero status, see log for more detail: 2

```

**Comment by sbrunner** [comment]
Isn't the file contribs/gmf/apps/mobile/js/controller.js included twice?

File: package.json (hunk 1 above)
Comment (fredj) [inline package.json]: We cannot use v3.14.1 because we need this commit: https://github.com/fredj/ol3/commit/916127d8f69eb163bf354ad8124a7797cbbdae98

**Comment by fredj** [comment]
The build error is now fixed, thanks @gberaudo

**Comment by tsauerwein** [comment]
LGTM

**Comment by sbrunner** [comment]
Looks good :-)

**Comment by fredj** [comment]
Thanks for the reviews

**Pull request merged**